code/src/components/TaskList.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import tasks from 'reducers/tasks'
+
+const TaskItem = styled.article`
+    border: 1px solid rgb(253, 93, 93);
+    border-radius: 5px;
+    padding: 1rem;
+    position: relative;
+    font-size: 0.8rem;
+`
+
+const DeleteButton = styled.button`
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    background: none;
+    border: none;
+    cursor: pointer;
+    font-size: 0.6rem;
+`
+
+const TaskList = () => {
+const taskList = useSelector((backpack) => backpack.tasks.items)
+
+// console.log(taskList)
```

Comment:
Easy to miss the console.logs (I do it all the time) but if you want I think you can remove this